client/src/components/new-request/search.vue

```diff
@@ -120,7 +126,7 @@
     </v-row>
 
     <v-row class="mt-4" no-gutters>
-      <v-col cols="5" v-if="isXproMras">
+      <v-col cols="4" v-if="isXproMras">
```

Comment:
You need to update the other v-col in this row to compensate for this change (line 144).

Reply:
ah you're right, will update